bin/run_cypress.py

```diff
@@ -0,0 +1,65 @@
+"""
+Run the Cypress tests
+"""
+import logging
+import os
+from pathlib import Path
+import shutil
+from subprocess import run, Popen, CalledProcessError, PIPE
+import sys
+
+logging.basicConfig()
+_log = logging.getLogger("run_cypress")
+_log.setLevel(logging.INFO)
+
+
+# noinspection SpellCheckingInspection
+def main():
+    cwd = Path(os.path.curdir).absolute()
+    script_dir = Path(__file__).parent
+    root_dir = script_dir.parent.absolute()
+    if root_dir != cwd:
+        _log.info(f"change directory to root {root_dir}")
+        os.chdir(root_dir)
+
+    npm_exe = shutil.which("npm")
+    waiton_exe = shutil.which("wait-on")
+    _log.info(f"executables: npm={npm_exe} wait-on={waiton_exe}")
+    assert npm_exe
+    assert waiton_exe
+
+    code = 0
+
+    _log.info("start flowsheet visualizer example")
+    fv_proc = Popen([npm_exe, "run", "startfv"], stderr=PIPE, stdout=PIPE)
+
+    _log.info("wait for flowsheet visualizer to start")
+    url = "http-get://127.0.0.1:49000/app?id=sample_visualization"
```

Comment:
This is mostly for my curiosity, but I don't think I've never seen the `http-get://` protocol. Is it a tool-specific thing?

Reply:
yes a cypress thing